src/Solnet.Rpc/SolanaRpcBatchWithCallbacks.cs

```diff
@@ -126,10 +196,23 @@ public void Flush()
                         KeyValue.Create("filters", filters),
                         HandleCommitment(commitment)));
 
-            _composer.AddRequest<List<AccountKeyPair>>("getProgramAccounts", parameters, callback);
```

Comment:
wow - so the type can be inferred by the compiler from the callback's type - that's very clever!